components/app/AppNav.jsx

```
@@ -166,6 +169,9 @@ export default function AppNav() {
                     <ThemeToggle />
                   </div>
                 </div>
+                <div className="mt-3 space-y-2">
+                  <StellarWalletIndicator />
+                </div>
                 <div className="mt-3">
                   <HeaderWalletStatus variant="mobile" />
```

ESLint

[error] 'HeaderWalletStatus' is not defined.

(react/jsx-no-undef)